Server/Game/unithelper.py

```diff
@@ -177,5 +141,181 @@ def setTeam(leader_ability, perks, units, username, version):
 
 	return err_msg
 
-def validateMove(unit, game, newX, newY):
+def calculateAttack(game, unit_dict, target):
+
+	# Create the response object
+	response = {}
+	response["Unit"] = unit_dict
+	response["Unit"]["ID"] = response["Unit"]["Unit"].id
+	response["Unit"].pop("Unit", None)
+	response["Target"] = {"ID":target.id}
+
+	return response
+
+def calculateHeal(game, unit_dict, target):
+
+	# Create the response object
+	response = {}
+	response["Unit"] = unit_dict
+	response["Unit"]["ID"] = response["Unit"]["Unit"].id
+	response["Unit"].pop("Unit", None)
+	response["Target"] = {"ID":target.id}
+
+	return response
+
+def updateValidAction(game, unit_dict, target_dict=None):
+	"""
+	Updates the game with the provided action
+
```

Comment:
Maybe we should rename this function?  An action isn't provided here and it seems like this mainly updates a unit's x and y location which for future actions may not be the thing we want to do.  Maybe we need to discuss what you exactly had in mind for this function?